cosmos/dbt/project.py

```diff
@@ -20,8 +20,15 @@ def create_symlinks(project_path: Path, tmp_dir: Path, ignore_dbt_packages: bool
             os.symlink(project_path / child_name, tmp_dir / child_name)
 
 
+def get_partial_parse_path(project_dir_path: Path) -> Path:
+    """
+    Return the partial parse (partial_parse.msgpack) path for a given dbt project directory.
+    """
+    return project_dir_path / DBT_TARGET_DIR_NAME / DBT_PARTIAL_PARSE_FILE_NAME
+
+
 def copy_msgpack_for_partial_parse(project_path: Path, tmp_dir: Path) -> None:
```

Comment:
Is this method getting used in our source code anymore? I could not find references apart from getting used in a test. If it's only for test maybe could move it there? Or do we want to keep it here since it has been a public method?

Reply:
Well spotted, its removed in 6401096e916c53f208a965f7dd7700a9d3c45bcc